Packs/Core/Integrations/CortexPlatformCore/CortexPlatformCore.py

```diff
@@ -14,6 +14,38 @@
 MAX_GET_INCIDENTS_LIMIT = 100
 SEARCH_ASSETS_DEFAULT_LIMIT = 100
 
+CASE_FIELDS = {
+    "case_id_list": "CASE_ID",
+    "case_domain": "INCIDENT_DOMAIN",
+    "case_name": "NAME",
+    "case_description": "DESCRIPTION",
+    "status": "STATUS_PROGRESS",
+    "severity": "SEVERITY",
+    "creation_time": "CREATION_TIME",
+    "asset_ids": "UAI_ASSET_IDS",
+    "asset_groups": "UAI_ASSET_GROUP_IDS",
+    "assignee": "ASSIGNED_USER_PRETTY",
+    "assignee_email": "ASSIGNED_USER",
+    "name": "CONTAINS",
+    "description": "DESCRIPTION",
+    "last_updated": "LAST_UPDATE_TIME",
+    "hosts": "HOSTS",
+    "starred": "CASE_STARRED",
+    "tags": "CURRENT_TAGS",
+}
+
+CASE_SEVERITY = {"low": "SEV_020_LOW", "medium": "SEV_030_MEDIUM", "high": "SEV_040_HIGH", "critical": "SEV_050_CRITICAL"}
```

Comment:
We already have severity mapping in this file, check if you can use it. 
SEVERITY_MAPPINGS